wrappers/python/c_files.cpp

```diff
@@ -139,6 +139,73 @@ void init_c_files(py::module &m) {
+        .def_readwrite("robot_mass", &rs2_safety_platform::robot_mass, "Robot Mass")
+        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_safety_platform, reserved, uint8_t, sizeof(rs2_safety_zone::reserved)), "Reserved");
+
+
+    py::class_<rs2_safety_zone> safety_zone(m, "safety_zone"); // No docstring in C++
+    safety_zone.def(py::init<>())
+        .def_readwrite("flags", &rs2_safety_zone::flags, "Flags")
+        .def_readwrite("zone_type", &rs2_safety_zone::zone_type, "Zone Type")
+        .def_property_readonly("zone_polygon", [](const rs2_safety_zone& self) { return reinterpret_cast<const std::array<sc_float2, sizeof(rs2_safety_zone::zone_polygon)>&> (self.zone_polygon);}, "zone polygon")
+        .def_readwrite("masking_zone_v_boundary", &rs2_safety_zone::masking_zone_v_boundary, "Masking Zone v Boundary")
+        .def_readwrite("safety_trigger_confidence", &rs2_safety_zone::safety_trigger_confidence, "Safety Trigger Confidence")
+        .def_readwrite("miminum_object_size", &rs2_safety_zone::miminum_object_size, "Miminum Object Size")
+        .def_readwrite("mos_target_type", &rs2_safety_zone::mos_target_type, "MOS Target Type")
+        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_safety_zone, reserved, uint8_t, sizeof(rs2_safety_zone::reserved)), "Reserved");
+
+    py::class_<rs2_safety_environment> safety_environment(m, "safety_environment"); // No docstring in C++
+    safety_environment.def(py::init<>())
+        .def_readwrite("grid_cell_size", &rs2_safety_environment::grid_cell_size, "Grid Cell Size")
+        .def_readwrite("safety_trigger_duration", &rs2_safety_environment::safety_trigger_duration, "Safety Trigger Duration")
+        .def_readwrite("max_linear_velocity", &rs2_safety_environment::max_linear_velocity, "Max Linear Velocity")
+        .def_readwrite("max_angular_velocity", &rs2_safety_environment::max_angular_velocity, "Max Angular Velocity")
+        .def_readwrite("payload_weight", &rs2_safety_environment::payload_weight, "Payload Weight")
+        .def_readwrite("surface_inclination", &rs2_safety_environment::surface_inclination, "Surface Inclination")
+        .def_readwrite("surface_height", &rs2_safety_environment::surface_height, "Surface Height")
+        .def_readwrite("surface_confidence", &rs2_safety_environment::surface_confidence, "Surface Confidence")
+        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_safety_zone, reserved, uint8_t, sizeof(rs2_safety_zone::reserved)), "Reserved");
+
+    py::class_<rs2_safety_preset> safety_preset(m, "safety_preset"); // No docstring in C++
+    safety_preset.def(py::init<>())
+        .def_readwrite("platform_config", &rs2_safety_preset::platform_config, "Platform Config")
+        .def_property_readonly("safety_zones", [](const rs2_safety_preset& self) { return reinterpret_cast<const std::array<rs2_safety_zone, sizeof(rs2_safety_preset::safety_zones)>&> (self.safety_zones);}, "Safety Zones")
```

Comment:
What does this mean? why readonly?

Reply:
putted this firstly to do some test, because could not convert the safety_zones array natively to python list
working on this, will update it asap.